api.js

```diff
@@ -108,6 +109,15 @@ Api.prototype._handleExceptions = function (data) {
 };
 
 Api.prototype._handleStats = function (stats) {
+	if (this.hasExclusive && !stats.hasExclusive) {
```

Comment:
Could you add a comment as to why the function is returning here? It's because if we're running exclusive tests, but the test file does not include exclusive tests, then we don't want those tests to be counted, right?